.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Is that file needed for running the certora workflow ? I'd guess not.

Reply:
It is not. I'll edit.